test/integration/contracts/mixins/with-events.spec.ts

```diff
@@ -117,6 +126,42 @@ describe('with-events mixin', () => {
 		);
 		assert(foo);
 
+		// Before enqueing up a create event we need to create a relationship to the random-slug type created above
```

Comment:
Same as other comment. Would it make more sense to update the types used in the test to match reality and remove the need to create a new relationship?